GLD/PHL/PHL_2017_LFS/PHL_2017_LFS_v01_M_v01_A_GLD/Programs/PHL_2017_GLD_LFS.do

```diff
@@ -1138,6 +1140,8 @@ foreach v of local ed_var {
 	tostring 	class ///
 				, format(`"%04.0f"') replace
 
+	replace 	class = "" 	if wave != "Q2" 	// only match second wave
```

Comment:
ensure that the only valid data is in the waves that have 4 digits. This is done for all iterations of isic and isco merges below